torchtext/experimental/vectors.py

```diff
@@ -0,0 +1,23 @@
+import torch
```

Comment:
You can find `download_from_url` and `extract_archive` func [Here](https://github.com/pytorch/text/blob/master/torchtext/utils.py).